### PR TITLE
Jordan Massingill

### PR DESCRIPTION
**Pull request by jordan-massingill**
Initial PR
Added dependencies and package.json

### CODE REVIEW

File: fake_index.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+server.get('/setname', (req, res) => {
+  req.session.name = 'Frodo';
+  res.send('got it');
+});
+
+server.get('/greet', (req, res) => {
+  const name = req.session.username;
+  res.send(`hello ${name}`);
+});
+
```
Comment (honda0306) [inline fake_index.js]: I can't remember if we've talked about how to conceal these before, but remember that it isn't safe to push any sort of token/hashing where the public can see. Let me know if you need help concealing them.

Comment (honda0306) [reply fake_index.js]: Referring to lines 119 && 120.

**Comment by honda0306** [comment]
Overall, your repo looks great! There are just a couple of `good practice` things that we can work on.

File: fake_index.js (hunk 1 above)
Comment (jordan-massingill) [reply fake_index.js]: This was a go-by I was using, it was Luis's code example from lecture. I meant to delete the file before doing any push's, but it seems as though I didn't!

**Pull request closed without merging**